yarn.lock

```diff
@@ -2,3765 +2,13932 @@
+    har-validator "~5.1.3"
+    http-signature "~1.2.0"
+    is-typedarray "~1.0.0"
+    isstream "~0.1.2"
+    json-stringify-safe "~5.0.1"
+    mime-types "~2.1.19"
+    oauth-sign "~0.9.0"
+    performance-now "^2.1.0"
+    qs "~6.5.2"
+    safe-buffer "^5.1.2"
+    tough-cookie "~2.5.0"
+    tunnel-agent "^0.6.0"
+    uuid "^3.3.2"
+
+"@cypress/xvfb@1.2.4":
+  version "1.2.4"
+  resolved "https://registry.yarnpkg.com/@cypress/xvfb/-/xvfb-1.2.4.tgz#2daf42e8275b39f4aa53c14214e557bd14e7748a"
+  integrity sha512-skbBzPggOVYCbnGgV+0dmBdW/s77ZkAOXIC1knS8NagwDjBrNC1LuXtQJeiN6l+m7lzmHtaoUw/ctJKdqkG57Q==
+  dependencies:
+    debug "^3.1.0"
+    lodash.once "^4.1.1"
+
+"@dhis2/app-adapter@4.0.0":
+  version "4.0.0"
+  resolved "https://registry.yarnpkg.com/@dhis2/app-adapter/-/app-adapter-4.0.0.tgz#738e726698f1eb7f6c56d4e51a657a8e549b5e34"
+  integrity sha512-NSU1LSZ4rc1v/czKT/IJRVptUOJz6QUbIKYjvadhF8JQCHfAIobOTQmnc9wJba2HAQDtgBMUftzfI6tvLT3TOQ==
+  dependencies:
+    "@dhis2/ui-widgets" "^2.0.4"
+    moment "^2.24.0"
 
-babel-plugin-syntax-trailing-function-commas@^6.3.13:
-  version "6.13.0"
-  resolved "https://registry.yarnpkg.com/babel-plugin-syntax-trailing-function-commas/-/babel-plugin-syntax-trailing-function-commas-6.13.0.tgz#2b84b7d53dd744f94ff1fad7669406274b23f541"
+"@dhis2/app-runtime@^2.0.4", "@dhis2/app-runtime@^2.1.1":
+  version "2.1.1"
+  resolved "https://registry.yarnpkg.com/@dhis2/app-runtime/-/app-runtime-2.1.1.tgz#442946bd52f1d4a968cf1ba05e68440f11f78996"
+  integrity sha512-jYf3YNiPE9RrAPD9C980s/3dhSh8JEq1s3B82O9T7gf+Vx2/covDIPnyU+64HmEjER+6gR5wHWbp7jOzBYSgLA==
 
-babel-plugin-transform-async-generator-functions@^6.17.0:
-  version "6.17.0"
-  resolved "https://registry.yarnpkg.com/babel-plugin-transform-async-generator-functions/-/babel-plugin-transform-async-generator-functions-6.17.0.tgz#d0b5a2b2f0940f2b245fa20a00519ed7bc6cae54"
+"@dhis2/app-shell@4.0.0":
+  version "4.0.0"
+  resolved "https://registry.yarnpkg.com/@dhis2/app-shell/-/app-shell-4.0.0.tgz#b628b87dc9b8216a6395eb83abedb8d8828fb907"
+  integrity sha512-Yd5Z6jFjaYqPKpcG07FQn6TcCHgCeCKXzHII0w1gPWN3KnDcQOY3CSgaOT9LpmPREjqwMMYKOu63xKQF+z7q4w==
   dependencies:
-    babel-helper-remap-async-to-generator "^6.16.2"
-    babel-plugin-syntax-async-generators "^6.5.0"
-    babel-runtime "^6.0.0"
-
-babel-plugin-transform-async-to-generator@^6.16.0:
-  version "6.16.0"
-  resolved "https://registry.yarnpkg.com/babel-plugin-transform-async-to-generator/-/babel-plugin-transform-async-to-generator-6.16.0.tgz#19ec36cb1486b59f9f468adfa42ce13908ca2999"
+    "@dhis2/app-adapter" "4.0.0"
+    "@dhis2/app-runtime" "^2.0.4"
+    "@dhis2/d2-i18n" "^1.0.5"
+    "@dhis2/ui-core" "^4.6.1"
+    classnames "^2.2.6"
+    moment "^2.24.0"
+    prop-types "^15.7.2"
+    react "^16.8.6"
+    react-dom "^16.8.6"
+    react-scripts "^3.3.0"
+    source-map-explorer "^2.1.0"
+    styled-jsx "^3.2.2"
+    typeface-roboto "^0.0.75"
+    typescript "^3.6.3"
+
+"@dhis2/cli-app-scripts@^4.0.0":
```

Comment:
Noticed this is using a slightly older version of the platform, doesn't have the Headerbar fixes for example - would be nice to upgrade it here